docs/spark/how-to-guides/windows-instructions.md

```diff
@@ -0,0 +1,253 @@
+     - Download [Apache Maven 3.6.0](http://mirror.metrocast.net/apache/maven/maven-3/3.6.0/binaries/apache-maven-3.6.0-bin.zip).
+     - Extract to a local directory e.g., `c:\bin\apache-maven-3.6.0\`.
+     - Add Apache Maven to your [PATH environment variable](https://www.java.com/en/download/help/path.xml) e.g., `c:\bin\apache-maven-3.6.0\bin`.
+     - Verify you are able to run `mvn` from your command-line.
+  5. Install **[Apache Spark 2.3+](https://spark.apache.org/downloads.html)**.
+     - Download [Apache Spark 2.3+](https://spark.apache.org/downloads.html) and extract it into a local folder (e.g., `c:\bin\spark-2.3.2-bin-hadoop2.7\`) using [7-zip](https://www.7-zip.org/).
+     - Add Apache Spark to your [PATH environment variable](https://www.java.com/en/download/help/path.xml) e.g., `c:\bin\spark-2.3.2-bin-hadoop2.7\bin`.
+     - Add a [new environment variable](https://www.java.com/en/download/help/path.xml) `SPARK_HOME` e.g., `C:\bin\spark-2.3.2-bin-hadoop2.7\`.
+     - Verify you are able to run `spark-shell` from your command-line.        
+        Sample console output:
+
+        ```
+        Welcome to
+              ____              __
+             / __/__  ___ _____/ /__
+            _\ \/ _ \/ _ `/ __/  '_/
+           /___/ .__/\_,_/_/ /_/\_\   version 2.3.2
+              /_/
+
+        Using Scala version 2.11.8 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_201)
+        Type in expressions to have them evaluated.
+        Type :help for more information.
+
+        scala> sc
+        res0: org.apache.spark.SparkContext = org.apache.spark.SparkContext@6eaa6b0c
+        ```
+
+        </details>
+
+  6. Install **[WinUtils](https://github.com/steveloughran/winutils)**.
+     - Download `winutils.exe` binary from [WinUtils repository](https://github.com/steveloughran/winutils). You should select the version of Hadoop the Spark distribution was compiled with, e.g. use hadoop-2.7.1 for Spark 2.3.2.
+     - Save `winutils.exe` binary to a directory of your choice e.g., `c:\hadoop\bin`.
+     - Set `HADOOP_HOME` to reflect the directory with winutils.exe (without bin). For instance, using command-line:
+
+       ```powershell
+       set HADOOP_HOME=c:\hadoop
+       ```
+
+     - Set PATH environment variable to include `%HADOOP_HOME%\bin`. For instance, using command-line:
+
+       ```powershell
+       set PATH=%HADOOP_HOME%\bin;%PATH%
+       ```
+
+Make sure you are able to run `dotnet`, `java`, `mvn`, `spark-shell` from your command-line before you move to the next section. Feel there is a better way? Please [open an issue](https://github.com/dotnet/spark/issues) and feel free to contribute.
+
+> [!NOTE]
+> A new instance of the command-line may be required if any environment variables were updated.
+
+## Build
+
+For the rest of the section, it is assumed that you have cloned the .NET for Apache Spark repository into your machine e.g., `c:\github\dotnet-spark\`.
```

Comment:
"It is assumed" might sound intimidating to new users (i.e. "if we're making assumptions, what else have I missed? Is the command below enough to clone the repo or are there other cloning steps I'm assumed to have done?").

```suggestion
For the remainder of this guide, you will need to have cloned the .NET for Apache Spark repository into your machine. You can choose any location for the cloned repository, e.g., `C:\github\dotnet-spark\`.
```

Reply:
Makes sense. Will make this change, thanks!